pages/index.vue

```
@@ -60,17 +79,153 @@ const cleanSearch = computed(() => {
   return getCleanText(search.value);
 });
 
+// Função para calcular distância entre duas coordenadas (em km)
+const calculateDistance = (lat1: number, lng1: number, lat2: number, lng2: number): number => {
```

ESLint

[error] 'calculateDistance' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)